client/src/components/related-items/relatedProducts/RelatedProductEntry.js

```diff
@@ -7,7 +7,8 @@ import ComparisonModal from '../comparisonModal/ComparisonModal';
 import changeProduct from '../../../redux/actions/changeProduct';
 import RatingStars from '../../RatingStars';
 
-const defaultImgae = 'https://images.squarespace-cdn.com/content/v1/5161eef2e4b05c308167a6fa/1555279090674-AG75CRP3YADFPC3XXUGO/ke17ZwdGBToddI8pDm48kN_ZoNdj1kv_gIvm4zjH76N7gQa3H78H3Y0txjaiv_0fDoOvxcdMmMKkDsyUqMSsMWxHk725yiiHCCLfrh8O1z4YTzHvnKhyp6Da-NYroOW3ZGjoBKy3azqku80C789l0jG2lbcDYBOeMi4OFSYem8AelHsSihC3tfiYK1eHEM7W3AVjJQSBul2wE-DqW7dygg/_MG_8028-1.jpg?format=1500w';
+const defaultImgae =
```

Comment:
This constant is named consistently through the file, so it's not breaking anything, but is the typo intentional? Might cause some headache later if you try to ctrl-f for "defaultImage" to find it.